catalog/nsili/catalog-nsili-transformer/src/main/java/org/codice/alliance/nsili/transformer/DAGConverter.java

```diff
@@ -658,37 +658,25 @@ private void addNsilVideoAttribute(MetacardImpl metacard, Node node) {
   }
 
   private void addNsilAssociation(List<Serializable> associations, Node node) {
-    switch (node.attribute_name) {
-      case NsiliConstants.IDENTIFIER:
-        associations.add(getString(node.value));
-        break;
-      default:
-        break;
+    if (node.attribute_name.equals(NsiliConstants.IDENTIFIER)) {
+      associations.add(getString(node.value));
     }
   }
 
   private String addNsilRelatedFile(MetacardImpl metacard, Node node) {
-    switch (node.attribute_name) {
-      case NsiliConstants.FILE_TYPE:
-        relatedFileType = getString(node.value);
-        if (StringUtils.isNotBlank(relatedFileUrl)) {
-          if (relatedFileType.equalsIgnoreCase(NsiliConstants.THUMBNAIL_TYPE)) {
-            metacard.setThumbnail(getThumbnail(relatedFileUrl));
-          }
-        }
-        break;
-      case NsiliConstants.URL:
-        relatedFileUrl = getString(node.value);
-        if (relatedFileType.equalsIgnoreCase(NsiliConstants.THUMBNAIL_TYPE)) {
-          if (StringUtils.isNotBlank(relatedFileUrl)) {
-            metacard.setThumbnail(getThumbnail(relatedFileUrl));
-          }
-        }
-        break;
-      default:
-        break;
+    if (node.attribute_name.equals(NsiliConstants.FILE_TYPE)) {
+      relatedFileType = getString(node.value);
+      if (NsiliConstants.THUMBNAIL_TYPE.equalsIgnoreCase(relatedFileType)
+          && StringUtils.isNotBlank(relatedFileUrl)) {
```

Comment:
❓ Isn't the `StringUtils.isNotBlank` check redundant since it cannot be blank if the first check is true?

Reply:
Is `relatedFileUrl` apart of the previous checks?